exercises/change/change_test.go

```diff
@@ -0,0 +1,68 @@
+package change
+
+import (
+	"reflect"
+	"testing"
+)
+
+const targetTestVersion = 1
+
+func TestTestVersion(t *testing.T) {
+	if testVersion != targetTestVersion {
+		t.Fatalf("Found testVersion = %v, want %v", testVersion, targetTestVersion)
+	}
+}
+
+func determineExpected(expected interface{}) (valid bool, list []int) {
+	_, isError := expected.(int)
+	if isError {
```

Comment:
Actually, I will use
    _, ok := expected.(int)
since ok is the typical bool result variable for a type assertion.